City-of-Helsinki/hel-fi-drupal-grants
# AU-574: Rework robot tests structure

# [AU-574](https://helsinkisolutionoffice.atlassian.net/browse/AU-574)


## What was done


Re-structured the robot tests to allow for larger test suite. 

Updated the robot test README, with instructions on how to run the new docker command and specific tests.

Also includes more tests, mainly in public Drupal website testing.

## How to test


* `make test-robot` 

There should be 3 tests, and all should pass.

[AU-574]: https://helsinkisolutionoffice.atlassian.net/browse/AU-574?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
